cli/version/version_match.go

```diff
@@ -0,0 +1,175 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/tarantool/tt/cli/util"
+)
+
+type requiredField uint8
+type requiredFields []requiredField
+
+const (
+	// Values according order of Fields in regex from `matchVersionParts()`.
+	requiredFieldBuildName requiredField = iota
+	requiredFieldMajor
+	requiredFieldMinor
+	requiredFieldPatch
+	requiredFieldReleaseType
+	requiredFieldReleaseNum
+	requiredFieldAdditional
+	requiredFieldHash
+	requiredFieldRevision
+
+	countOfRequiredFields
+)
+
+// NotFoundError is sentinel error for case when required version not found.
+type NotFoundError struct {
+	expected string
+}
+
+// Error - error interface implementation for NotFoundError.
+func (e NotFoundError) Error() string {
```

Comment:
Is is usual something like:

https://pkg.go.dev/bytes#Reader.Read
```suggestion
// Error implements the [error] interface.
func (e NotFoundError) Error() string {
```

Reply:
Done.